src/main/java/com/google/firebase/auth/UserRecord.java

```diff
@@ -499,6 +512,12 @@ public UpdateRequest setCustomClaims(Map<String,Object> customClaims) {
       return this;
     }
 
+    @VisibleForTesting
```

Comment:
It's visible for functional requirements (not just testing). So the annotation can be removed.